packages/vhd-lib/Vhd/VhdAbstract.js

```diff
@@ -334,4 +334,25 @@ exports.VhdAbstract = class VhdAbstract {
     stream.length = footer.currentSize
     return stream
   }
+
+  /*
+   * check if all the data of a child are already contained in this vhd
+   */
+
+  async contains(child) {
+    await this.readBlockAllocationTable()
+    await child.readBlockAllocationTable()
```

Comment:
I believe this method (and some others), are pitfalls, and should be implcitely called when necessary (e.g., when calling blocks).

If you agree, I'll do the change.